IdentityCore/tests/MSIDLoggerTests.m

```diff
@@ -33,6 +33,7 @@ @interface MSIDLoggerConnectorMock : NSObject <MSIDLoggerConnecting>
 @property (nonatomic) BOOL piiLoggingEnabledValue;
 @property (nonatomic) BOOL shouldLogValue;
 @property (nonatomic) BOOL sourceLineLoggingEnabledValue;
+@property (nonatomic) BOOL loggingQueueEnabledValue;
```

Comment:
Can we add a unit test with logger queue enabled == NO?

Reply:
added